setup.py

```diff
@@ -29,6 +29,7 @@
         "sphinx>=6.0.0",
         "sphinx_rtd_theme>=1.0.0",
         "towncrier>=24,<25",
+        "grpcio>=1.41.0",
```

Comment:
It would be best to have only one location for a dependency to be listed. You can remove it from `docs` and add the following to `tox.ini`:

```
[testenv:docs]
extras=.
```

This will install everything defined within setup.py's `install_requires` when running the docs job. I thought that had already been done, but I don't see it in `main`, so not sure.